pkg/processor/runtime/python/py/wrapper_common.py

```diff
@@ -45,11 +45,24 @@ class WrapperFatalException(Exception):
     pass
 
 
+class EventSocketDisconnected(Exception):
+    """
+    Signals about Event Socket disconnect (fatal for sync wrapper, ok for async)
```

Comment:
why is it not fatal for async?
the same `_resolve_event_message_length` function is used.

Reply:
@TomerShor good question! Because if something happens with one socket in async, we should not fail whole wrapper, just close this specific connection and that's it